presto-main/src/main/java/com/facebook/presto/sql/analyzer/FeaturesConfig.java

```diff
@@ -957,6 +958,18 @@ public FeaturesConfig setMultimapAggGroupImplementation(MultimapAggGroupImplemen
         return this;
     }
 
+    public boolean isTDigestEnabled()
```

Comment:
I'm wondering why do we choose a static config rather than a session property. Do we worry registering these functions can cause trouble? If not why don't we just register them, and depending on a session property decide whether to use it or not?